Utils/Functions/Net/Download/script.js

```diff
@@ -32,6 +32,10 @@ var Downloader = function () {
         that._localFile = localFile;
         return that;
     };
+    that.ifUpdateAvailable = function (ifUpdateAvailable) {
```

Comment:
I'm a little confused about the type of the variable `ifUpdateAvailable` that should be passed to the function.
When taking a look at normal java APIs, like https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html I normally think that methods beginning with an `if` take a lambda function,  but I believe your method takes a boolean right?